Proposed/asc047.md

```diff
@@ -0,0 +1,167 @@
+#### **Title**
+New instruction: `spe`
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+Varun Kanwar <varunkanwar1@gmail.com>
+
+#### **Motivation**
+Solid phase extraction (SPE) is a sample preparation technique by which compounds dissolved or suspended in a liquid mixture (mobile phase) are isolated and purified according to their physical and chemical properties. SPE involves passing a liquid matrix through an adsorbent solid (stationary phase), wherein compounds of interest are separated from present interferents by their differential affinities for the stationary phase. This process is routinely used to extract analytes from a wide variety of liquid matrices.
+
+The process typically begins with cartridge `conditioning` to wet and settle the sorbent bed and to remove fine residue. The cartridge subsequently undergoes an `equilibration` where it is treated with a matrix-like solvent (this step can be skipped if the conditioning solvent is matrix-like). The matrix containing compound(s) of interest is then loaded on to the column, `rinse`d with solvents to elute interferents, after which a final `elution` for the target compound(s) can take place.
+
+#### **Proposal**
+We propose the addition of an `spe` instruction to Autoprotocol to represent the separation of a liquid sample into multiple eluted fractions via solid phase extraction.
+
+```
+{
+  "op": "spe",
+  /* Source aliquot with liquid matrix */
+  "object": Well,
+  /* Sorbent cartridge used to extract the target compound. Required */
+  "cartridge": String or ColumnType,
+  /*
+   * The direction of pressure applied to the cartridge to force liquid flow.
+   * Positive pressure is applied from above the cartridge as a pushing force,
+   * whereas negative pressure is applied from below the cartridge as vacuum.
+   */
+  "pressure_mode": Enum("positive", "negative")
+  /* Containers used to collect eluates. Required */
+  "elution_containers": List(Container)
+  /*
+   * Parameters for conditioning the cartridge with one or more solvents.
+   * Optional
+   */
+  "conditioning": List(
```

Comment:
use the same tense for all the mobile phase steps, eg. 
`condition`, `equilibrate`, `load_sample`, `rinse`, `elute`